sunpy/net/jsoc/attrs.py

```diff
@@ -71,6 +72,41 @@ def __init__(self, value):
         self.value = value
 
 
+class Cutout(DataAttr):
+    """
+    Select a cutout region to be processed by JSOC
+
+    Parameters
+    ----------
+    bottom_left
+    width
+    height
+    tracking
+    register
+    nan_off_limb
+    """
+
+    def __init__(self, bottom_left, width, height, tracking=False, register=False,
```

Comment:
You should use the get rectangle helper here I think: https://github.com/sunpy/sunpy/blob/master/sunpy/coordinates/utils.py#L269

Reply:
I've add the rectangle helper, though I'm not totally sure I'm doing it right. Looking at `GenericMap.submap`, there seems to be a lot of additional input validation outside of the helper.